server/.env.SAMPLE

```diff
@@ -22,4 +22,9 @@ COMPANY_NAME = ""
 
 # Key for Open Weather API
 WEATHER_API_KEY = ""
-WEATHER_API_CITY_ID = ""
\ No newline at end of file
+WEATHER_API_CITY_ID = ""
+
+# Git
```

Comment:
Server crashes on empty configs - please correct this. 
A correct .env file with the right config keys but empty values should not cause the server to crash.